src/main/java/Main.java

```diff
@@ -1,114 +1,188 @@
 import java.util.Scanner;
 
-public class Main{
+public class Main {
     public static final short MAX_PTS_VIE = 100;
     public static final short PTS_BOUCLIER = 25;
     public static final short MAX_ATTAQUE_ENNEMI = 5;
     public static final short MAX_VIE_ENNEMI = 30;
     public static final short MAX_ATTAQUE_JOUEUR = 5;
     public static final short REGENARATION_BOUCLIER_PAR_TOUR = 10;
     public static String nomPersonnage;
-    public static short ptsDeVie;
-    public static short ptsBouclier;
+    public static short ptsDeVieJoueur;
+    public static short ptsVieEnnemis;
+    public static short ptsBouclierJoueur;
     public static short nbEnnemisTues;
-    public static boolean bouclierActif = true;
+    public static boolean bouclierActifJoueur = true;
+    public static short attaque;
+    public static String continuerPartie;
+    public static short nbEnnemis;
 
-    public static void main(String[] args){
+
+    public static void main(String[] args) {
+        //************ Début évaluation **************
+        //Initialiser le personnage du joueur
         initPersonnage();
+        //Initialiser les "ennemis", tableau "ennemis" avec "nbEnnemis" et nb vie ennemis => ptsVieEnnemi
+        initEnnemis();
+        //Initialiser les combats => tant que ennemis il y a des combats
+        // Combat avec le 1er ennemi
+        for (short [] ptsVieEnnemis; nbEnnemis > 0;) {
```

Comment:
Il te manque la récupération de la sortie de la fonction `initEnnemis` qui te permets justement d'obtenir un tableau de `short[]` représentant les points de vie des ennemis à combattre. Il est préférable d'utiliser un `while` au lieu d'un `for` si tu n'as pas d'incrémentation.
```suggestion
        short [] ptsVieEnnemis = initEnnemis();
        //Initialiser les combats => tant que ennemis il y a des combats
        // Combat avec le 1er ennemi
        while(nbEnnemis > 0) {
```